### ISSUE TITLE
[pt] Punctuation suggestion withou effect

### ISSUE THREAD

**Issue by ricardojosehlima** [languagetool-org/languagetool#7297]
Hi,
Let's take a look at the attached picture
![Captura de tela de 2022-10-17 07-43-42](https://user-images.githubusercontent.com/36453534/196203451-abcfd750-f6bd-4c3e-9028-ad41d0d8ac57.png)
It says I must put a space after ; (it happens with : too). However, there's already a space there. And when I accept the rule suggestion... It keeps saying that I must put a space...

**Comment by marcoagpinto** [comment]
Ahhhh… I will have a look tonight or at 5am.

I can even guess how to fix it… 😄 😄 😄 😄 😄 😄 😄 

An antipattern with two or three lines of XML… 😄 😄 😄 😄 😄 

❤️ ❤️ ❤️ ❤️ ❤️

**Comment by marcoagpinto** [comment]
Ohhhhh… I was just reflecting… I believe I saw something similar years ago… maybe it is a bug that has been around for years?

**Comment by ricardojosehlima** [comment]
Hmm maybe not, I don't recall being corrected in these cases recently, and I use " after : and ; frequently. Or perhaps I didn't notice it!?

**Comment by marcoagpinto** [comment]
@ricardojosehlima 

I can't reproduce it with the sentence:
`E (originalmente); “A noiva casa de branco”; falar.`

Could you type here a full sentence where it happens?

Thanks!

**Comment by ricardojosehlima** [comment]
Try this in the attached file.
![Captura de tela de 2022-10-17 15-59-17](https://user-images.githubusercontent.com/36453534/196260132-fb33638e-f18d-48ed-bd7c-5adc9712a6e1.png)
Strangely, it mus be exactly as it is... If you leave falar de Expressões Regulares e mostrar exemplos; mostrar tabelas com testes.
 out, you'll see no errors flagged...

**Comment by marcoagpinto** [comment]
It doesn't trigger the error:

`→ Sugestões: "comigo" (classificado com substantivo originalmente); "A noiva casa de branco"; falar de Expressões Regulares e mostrar exemplos; mostrar tabelas com testes.`

It only flags the "s not the spaces.

Maybe it is the bullet (arrow) at the start of the sentence that affects it?

**Comment by ricardojosehlima** [comment]
Indeed, it happened in Google Docs. In the e-mail, no error flagged. Maybe the arrow...

**Comment by marcoagpinto** [comment]
Ahhhh… there is only one rule in the grammar.xml with the text:
`<message>Deve colocar um espaço após '\1'.</message>`

I will try to generate a 0_before.txt and then create an antipattern for ":" and ";" and quotes and do a 1_after.txt.

If it removes hits, it means it is fixed.

This is the best I can do.

**Comment by marcoagpinto** [comment]
@ricardojosehlima 

This one is hard as it doesn't produce a single hit in 600 000 sentences, so I am unable to check the fix:
https://github.com/languagetool-org/languagetool/commit/b6d6120767e0dfbace673121819ddd88bc8b548d

@danielnaber 
Daniel, is there the possibility of adding to the browsers add-on information regarding when the on-line database has been updated (date) so that one may know whether a fix is already working or not?

Thanks!
